server/routes/changeSupportingPrison/changePrisonConfirmation.test.ts

```diff
@@ -56,10 +56,10 @@ afterEach(() => {
   jest.resetAllMocks()
 })
 
-describe('GET /confirm-change', () => {
+describe('GET /', () => {
```

Comment:
changed to match the remove functionality as this is the primary action within the flow